src/Tribe/Aggregator/Cron.php

```diff
@@ -401,33 +403,8 @@ public function start_batch_pushing_records() {
 			return;
 		}
 
-		$records = Tribe__Events__Aggregator__Records::instance();
-
-		$query = $records->query(
-			[
-				'post_status'    => Tribe__Events__Aggregator__Records::$status->pending,
-				'posts_per_page' => 250,
-				'order'          => 'ASC',
-				'meta_query'     => [
-					'origin-not-csv'               => [
-						'key'     => '_tribe_aggregator_origin',
-						'value'   => 'csv',
-						'compare' => '!=',
-					],
-					'batch-push-support-specified' => [
-						'key'     => '_tribe_aggregator_allow_batch_push',
-						'value'   => true,
-						'compare' => '=',
-					],
-					'batch-not-queued'             => [
-						'key'     => '_tribe_aggregator_batch_started',
-						'value'   => 'bug #23268',
-						'compare' => 'NOT EXISTS',
-					],
-				],
-				'after'          => '-4 hours',
-			]
-		);
+		$records = Records::instance();
+		$query = $this->get_batch_pushing_records();
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: ff2efb3a2e02a2a93f28515a3aef6e7a</sub>